<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>promoTimerClass</class>
 <widget class="QMainWindow" name="promoTimerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>promoTimer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="breakLength">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sesionLength">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timerText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="promotimer.qrc"/>
 </resources>
 <connections/>
</ui>
